restli-server/src/main/java/com/linkedin/restli/internal/server/model/RestLiAnnotationReader.java

```diff
@@ -2595,6 +2595,13 @@ private static void validateFinderMethod(final ResourceMethodDescriptor finderMe
     Method method = finderMethodDescriptor.getMethod();
     Class<?> valueClass = resourceModel.getValueClass();
 
+    if (valueClass == null)
+    {
+      throw new ResourceConfigException("@Finder method '" + method.getName()
+          + "' on class '" + resourceModel.getResourceClass().getName()
+          + "' has an invalid return type, The return type must be a RecordTemplate'");
```

Comment:
nits:
1. either make the `T` lowercase or replace the preceding comma with a period.
2. remove the dangling `'` at the end of the message.